template-for-repository/proofs/Makefile-project-defines

```diff
@@ -13,6 +13,12 @@
 # Makefile.common.
 ################################################################
 
+# Name of this proof project, displayed in proof reports
+# PROJECT_NAME =
+
+# Path to litani executable
+# LITANI =
```

Comment:
Same question: Are you expecting this to live in a top-level file for the whole repository?

Reply:
Resolved during offline conversation, and added a clarifying comment.